docs/quickstart.md

```diff
@@ -0,0 +1,91 @@
+## Installation
+
+`@angular-redux/store` has a peer dependency on redux, so we need to install it as well.
+
+```sh
+npm install --save redux @angular-redux/store
+```
+
+## Quick Start
+
+```typescript
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { AppModule } from './containers/app.module';
+
+platformBrowserDynamic().bootstrapModule(AppModule);
+```
+Import the `NgReduxModule` class and add it to your application module as an
+`import`. Once you've done this, you'll be able to inject `NgRedux` into your
+Angular components. In your top-level app module, you
+can configure your Redux store with reducers, initial state,
+and optionally middlewares and enhancers as you would in Redux directly.
+
+```typescript
+import { NgReduxModule, NgRedux } from '@angular-redux/store';
+import reduxLogger from 'redux-logger';
+import { rootReducer } from './reducers';
+
+interface IAppState { /* ... */ };
```

Comment:
Yeah I'll add a note. However I didn't want to get into too complicated an example in this tutorial. I might write a second chapter later.